src/Orchard.Web/Modules/Orchard.Tokens/Providers/RequestTokens.cs

```diff
@@ -1,25 +1,30 @@
 ﻿using System;
+using System.Linq;
 using System.Web;
 using Orchard.ContentManagement;
 using Orchard.Localization;
+using System.Globalization;
+using System.Text.RegularExpressions;
 
 namespace Orchard.Tokens.Providers {
     public class RequestTokens : ITokenProvider {
         private readonly IWorkContextAccessor _workContextAccessor;
         private readonly IContentManager _contentManager;
+        private static string[] _textChainableTokens;
 
         public RequestTokens(IWorkContextAccessor workContextAccessor, IContentManager contentManager) {
             _workContextAccessor = workContextAccessor;
             _contentManager = contentManager;
+            _textChainableTokens = new string[] { "QueryString", "Form" };
             T = NullLocalizer.Instance;
         }
 
         public Localizer T { get; set; }
 
         public void Describe(DescribeContext context) {
             context.For("Request", T("Http Request"), T("Current Http Request tokens."))
-                .Token("QueryString:*", T("QueryString:<element>"), T("The Query String value for the specified element."))
-                .Token("Form:*", T("Form:<element>"), T("The Form value for the specified element."))
+                .Token("QueryString:*", T("QueryString:<element>"), T("The Query String value for the specified element. If you want to chain text, surround the <element> with brackets [e.g. QueryString:(param1)]."))
+                .Token("Form:*", T("Form:<element>"), T("The Form value for the specified element. If you want to chain text, surround the <element> with brackets [e.g. Form:(param1)]."))
```

Comment:
I think these explanation strings are a bit confusing, as they actually contain brackets but what I think you are referring to are the parentheses. A better hint text in my opinion would be to remove the brackets around the example and use the word paretheses, like so:

`The query string value for the specified element. To chain text, surround the <element> with parentheses, e.g. 'QueryString:(param1)'.`